docs/service-brokers/docs/002-overview-azure-broker.md

```diff
@@ -12,3 +12,7 @@ See the details of each Service Class and its specification in the Service Catal
 For more information about the Service Brokers, see the Service Brokers **Overview** document.
 
 >**NOTE:** Kyma uses the Microsoft Azure Broker open source project. To ensure the best performance and stability of the product, Kyma uses a version of the Azure Broker that precedes the newest version released by Microsoft.
+
+Kyma provides Namespace-scoped Azure Broker. In each Namespace, you can configure the Azure Broker against different subscriptions. Install the Azure Broker by provisioning the **Azure Broker** class provided by the Helm Broker.
```

Comment:
Make it similar to documentation about GCP broker overview, see: https://github.com/kyma-project/kyma/blob/c52ff7150cad9e3aa8e4a397823a87e2a1c658cd/docs/service-brokers/docs/003-overview-gcp-broker.md
- for example, add same nice diagrams

Reply:
Explicitly say that new broker deployment will be created per namespace.